server/tso/allocator_manager.go

```diff
@@ -309,11 +309,7 @@ func (am *AllocatorManager) SetUpAllocator(parentCtx context.Context, dcLocation
 	switch dcLocation {
 	// For Global TSO Allocator
 	case config.GlobalDCLocation:
-		// Because Global TSO Allocator only depends on PD leader's leadership,
-		// so we can directly initialize it here.
-		if err := am.mu.allocatorGroups[dcLocation].allocator.Initialize(0); err != nil {
-			return err
-		}
+		return nil
```

Comment:
Why is it no longer possible to return err?

Reply:
In the previous code, the initialization of the Global TSO Allocator is carried out with `SetUpAllocator()`. Now it has been moved to `campaignLeader()`, so there is no other work to be done here, just return `nil`.